src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -1396,6 +1396,11 @@ internal async Task<bool> StartTaskHubWorkerIfNotStartedAsync()
                         await this.defaultDurabilityProvider.CreateIfNotExistsAsync();
                         await this.taskHubWorker.StartAsync();
 
+                        if (this.Options.StoreInputsInOrchestrationHistory == true)

```

Comment:
In C#, when testing `bool` values in an `if` statement, it's expected that you don't explicitly compare it to `true` or `false`. Instead, you do either `if (value) {...}` or `if (!value) {...}`. See line 1386 above for an example. For this line, let's remove the `== true` so that you are being consistent with normal C# code style.

```csharp
if (this.Options.StoreInputsInOrchestrationHistory)
```